### PR TITLE
fix(chrome): add background-size to display icons

### PR DESCRIPTION
**Pull request by mohamedMok**
## I have read [the contributing guidelines](https://mozaic.adeo.cloud/Contributing/)

- [ ] Yes
- [ ] No

## Does this PR introduce a [breaking change](https://mozaic.adeo.cloud/Contributing/Developers/GitConventions/#breaking-changes-)?

- [ ] Yes
- [ ] No

## Describe the changes

GitHub issue number or Jira issue URL: N/A

### CODE REVIEW

**Comment by mohamedMok** [comment]
Logo
**Checkmarx One – Scan Summary & Details** – [03afb71f\-97a4\-4db4\-af62\-4b4afc865b72](https://eu.ast.checkmarx.net/projects/d2ea2837-0ed1-4404-8c6a-f1a1376ffd05/scans?id=03afb71f-97a4-4db4-af62-4b4afc865b72&branch=fix-chrome-icons)
### No New Or Fixed Issues Found

File: packages/styles/components/_c.notification.scss [hunk 1]
```diff
@@ -103,6 +103,7 @@
 
       background: transparent
         url(inline-icons('cross-16', $notification-close-icon-color)) no-repeat;
+      background-size: contain;
```
Comment (tiloyi) [inline packages/styles/components/_c.notification.scss:106]: ~~background-size: contain;~~ => `background-size: $mu100;`

File: packages/styles/components/_c.toggle.scss [hunk 2]
```diff
@@ -73,6 +73,7 @@
       background: $color-toggle-off-circle
         url(inline-icons("control-cross-16", $color-toggle-off-background))
         no-repeat center;
+      background-size: contain;
```
Comment (tiloyi) [inline packages/styles/components/_c.toggle.scss:76]: ~~background-size: contain;~~ => `background-size: $mu100;`

File: packages/styles/components/_c.toggle.scss [hunk 3]
```diff
@@ -83,6 +84,7 @@
           "notification-available-16",
           $color-toggle-on-background
         ));
+        background-size: contain;
```
Comment (tiloyi) [inline packages/styles/components/_c.toggle.scss:87]: ~~background-size: contain;~~ => Due to the "cascade", it is not necessary to repeat this value here.

File: packages/styles/components/_c.toggle.scss [hunk 4]
```diff
@@ -93,6 +95,7 @@
           "control-cross-16",
           $color-toggle-disabled-background
         ));
+        background-size: contain;
```
Comment (tiloyi) [inline packages/styles/components/_c.toggle.scss:98]: ~~background-size: contain;~~ => Due to the "cascade", it is not necessary to repeat this value here.

File: packages/styles/components/_c.toggle.scss [hunk 5]
```diff
@@ -103,6 +106,7 @@
           "notification-available-16",
           $color-toggle-disabled-checked-background
         ));
+        background-size: contain;
```
Comment (tiloyi) [inline packages/styles/components/_c.toggle.scss:109]: ~~background-size: contain;~~ => Due to the "cascade", it is not necessary to repeat this value here.

**Review by tiloyi — APPROVED** [review]

**Pull request merged**